directus-aws/index.md

```diff
@@ -0,0 +1,206 @@
+
+Add a name for the server called `Directus Server` and select the Amazon Linux image (you can also choose another image to suit your needs).
+
+![An AWS EC2 creation page with Amazon Linux image selected](ec2-image.png)
+
+Create a key pair that you can use for logging in to the EC2 instance.
+
+Also, for network settings, select the existing security group and choose the default security group for connecting to the EC2 instance.
+
+![An AWS EC2 creation page with options to configure the  VPC security group](vpc.png)
+
+For storage options, the default selection meets the requirements for running a Directus instance.
+
+![An AWS EC2 creation page with options to configure the  Network settings](ec2-network.png)
+
+Click on the `Launch Instance` button, and this will create a new EC2 instance for you
+
+### Set up permission
+
+To ensure that the EC2 instance can be accessed and operated from anywhere, head over to the security group in your EC2 instance and add the following inbound rules:
+
+| Name  | Security group rule ID | IP version | Type | Protocol | Port Range | Source |
+| --- | --- | --- | --- | --- | --- | --- |
+| -  | your default security group | IPv4 | SSH | TCP | 22 | 0.0.0.0/0 |
+| -  | your default security group | IPv4 | HTTP | TCP | 80 | 0.0.0.0/0 |
+| -  | your default security group | IPv4 | HTTPS | TCP | 443 | 0.0.0.0/0 |
+
+![EC2 Network security settings inbound rules](inbound.png)
+
+Also, create an outbound rule for connecting to the database:
+
+| Name  | Security group rule ID | IP version | Type | Protocol | Port Range | Source |
+| --- | --- | --- | --- | --- | --- | --- |
+| -  | your default security group | IPv4 | PostgreSQL | TCP | 5432 | 0.0.0.0/0 |
+
+![EC2 Network security settings outbound rules](outbound.png)
+Save these rules.
+
+## Set up Directus on AWS EC2
+
+On the EC2 instance page, click on the connect button to connect to your AWS EC2 instance using EC2 Instance Connect (you can also connect to EC2 via other methods such as a session manager or SSH client).
+
+![EC2 instance connect page](ec2-connect.png)
+
+Once connected, run the command to install Docker on your EC2 server:
+
+```bash
+sudo yum install -y docker
+```
+
+Next, install Docker-compose with the command:
+
+```bash
+sudo curl -L https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
+```
+
+Grant permission to run the `docker-compose` command without using the `sudo` command
+
+```bash
+sudo chmod +x /usr/local/bin/docker-compose
+```
+
+Start the docker service with the command:
+
+```bash
+sudo service docker start
+```
+
+Create a new directory called `directus` and navigate into the directory with the command:
+
+```bash
+mkdir directus && cd directus
+```
+
+In the `directus` directory, create a `docker-compose.yml` file and update its content using `vim` with the command:
+
+```bash
+touch docker-compose.yml && vim docker-compose.yml
```

Comment:
Please replace with `nano` - no point trapping people in a text editor. Please make sure to do this throughout.